bdk-ffi/src/lib.rs

```diff
@@ -1419,4 +1612,142 @@ mod test {
         assert!(tx_builder_result.psbt.fee_amount().is_some());
         assert_eq!(tx_builder_result.psbt.fee_amount().unwrap(), 220);
     }
+
+    #[test]
+    fn test_descriptor_templates() {
```

Comment:
Need to remove or at least comment out the `println!` lines and add back in the `assert!` lines at the end.

Reply:
Oh forgot about the issue with bitcoindevkit/bdk#817, ok to leave the asserts commented out until we update to bdk 0.26 which should fix the issue.